### PR TITLE
Add missing MSP_SET_OSD_CANVAS support

### PR DESCRIPTION
**Pull request by haslinghuis**
Activating

https://github.com/betaflight/betaflight/blob/1e093159658d889eec49abf90c2802296b36ba78/src/main/msp/msp.c#L4312-L4331

to set `displayPortDevice` to `MSP` automatically

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Comment by haslinghuis** [comment]
@SteveCEvans please confirm

**Review by SteveCEvans — APPROVED** [review]

**Pull request merged**
